src/pages/Settings.module.scss

```diff
@@ -0,0 +1,12 @@
+.settings-wrapper {
+  display: flex;
+  flex-direction: column;
+  gap: 1em;
```

Comment:
Just nitpicking here, but I'm using `rem` to try to keep it consistent everywhere. Is there an advantage in using `em`?

Reply:
it scales with the current font-size but either is fine. I'm just more used to `em`